### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by edutjie**

### CODE REVIEW

**Review by codewithdjaeger — APPROVED** [review]

**Review by codewithdjaeger — CHANGES_REQUESTED** [review]
- Please change this file name, `app/components/landing-page/OurStories/utils/[contents].tsx`, because the use of `[]` is only for dynamic file naming

File: app/components/landing-page/OurStories/index.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+					<Link href='/ourstories2'>
+						<Story>
+							<h3 className='w-full'>TITLE</h3>
+						</Story>
+					</Link>
+					<Link href='/ourstories3'>
+						<Story>
+							<h3 className='w-full'>TITLE</h3>
+						</Story>
+					</Link> */}
```
Comment (codewithdjaeger) [inline app/components/landing-page/OurStories/index.tsx:56]: Will it be used again?

File: app/components/landing-page/OurStories/interfaces/StoryProps.interface.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export interface StoryProps {
```
Comment (codewithdjaeger) [inline app/components/landing-page/OurStories/interfaces/StoryProps.interface.tsx:1]: Please change the file name format into `[component].interface.tsx` and the class name into [Component]Interface

File: app/components/landing-page/Messages/index.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+		<section className='h-full text-white'>
+			<div className='relative'>
+				<img src='/assets/messages-petikdua.svg' className='mt-5 -mb-36 w-20' />
+				<h2 className='my-24 text-center 2sm:max-w-md 2sm:text-left pt-3 pl-14'>
+					Messages
+				</h2>
+			</div>
+			<div className='2md:grid grid-rows-2 grid-cols-2 gap-7 text-gray-900 xl:max-w-3/4 mx-auto 2md:-mt-36 items-center'>
+				<MessagesCard className='row-span-2'>
+					{/* <p className='max-w-xs text-center 2sm:max-w-4xl 2sm:text-left'> */}
```
Comment (codewithdjaeger) [inline app/components/landing-page/Messages/index.tsx:15]: What is this comment for?

**Review by codewithdjaeger — CHANGES_REQUESTED** [review]
Next Time, Please do not push commented code without any information provided

File: app/components/landing-page/Messages/index.tsx (hunk 3 above)
Comment (edutjie) [reply app/components/landing-page/Messages/index.tsx:15]: ohh itu code yang lama lupa apus 🤦‍♂️

**Review by codewithdjaeger — APPROVED** [review]

**Pull request merged**